outputs/reports/skills_extraction.md

```diff
@@ -69,6 +70,15 @@ A summary of the experiments with training the model is below.
 | 20220630          | blank en       | 180             | 45              | 50                   | 0.3           | 0.001         | True                |                                                                                                  | 0.49     | 0.39          | 0.48   |                       |
 | 20220629          | blank en       | 156             | 39              | 50                   | 0.3           | 0.001         | True                |                                                                                                  | 0.52     | 0.45          | 0.51   |                       |
 
+More in-depth metrics for `20230808`:
+
+| Entity     | F1    | Precision | Recall |
+| ---------- | ----- | --------- | ------ |
+| Skill      | 0.612 | 0.712     | 0.537  |
+| Experience | 0.524 | 0.647     | 0.441  |
+| Benefit    | 0.531 | 0.708     | 0.425  |
```

Comment:
I know the recall isn't ideal but honestly impressed with the precision given how few data points are labelled benefit

Reply:
yeh it's good! And really this metric should be just applied to the prodigy test data too, I think if it was then perhaps the precision would be higher (think the recall would be the same though)?

I had such a pain with this because I wanted to train one NER model to predict all the entity types, but obviously only the most recent labelled job adverts had BENEFIT labels. So, the model effectively sees the label-studio training data as saying there are no BENEFITs in when there probably are.

Couldn't find a way to tell it to only train the BENEFIT label on the new data, whilst using all the data to train the SKILL and EXPERIENCE labels.

I tried training a model just with the prodigy data and on the BENEFIT label it got:

|F1 | Precision | Recall | 
|---|---|---|
0.539 | 0.655 | 0.458

so arguably more or less the same!


FYI I made a table with the training experiments [here](https://docs.google.com/spreadsheets/d/1zilKKB7wa487ufOJvaFETNYXs7bfwGrnegj2u0hO8bE/edit#gid=0)